nbgitpuller/handlers.py

```diff
@@ -53,7 +53,7 @@ def get(self):
         try:
             repo = self.get_argument('repo')
             branch = self.get_argument('branch')
-            repo_dir = repo.split('/')[-1]
+            repo_dir = self.get_argument('repodir')
```

Comment:
Should this be:
```suggestion
            repo_dir = self.get_argument('targetPath')
```
instead?

Reply:
I made your change, but the variable `repodir` is passed around and required further changes. To make things clearer I've replaced `repodir` with `targetpath` throughout `handlers.py`, `index.js` and `status.html`